torch/nn/quantized/modules/conv.py

```diff
@@ -17,6 +17,11 @@
 from torch.nn.quantized.modules.utils import _quantize_weight
 from torch.nn.utils import fuse_conv_bn_weights
 
+try:
+    from typing_extensions import Final
+except ImportError:
+    from torch.jit import Final
```

Comment:
Cant we define squeeze_dim to be an integer? Is this the recommended way to define constant integers for scripting?

Reply:
According to the documentation, it is: https://pytorch.org/docs/stable/jit.html#constants

Reply:
I think we should just import from torch.jit. why are you importing from typing_extensions?

Reply:
oh, I saw that in documentation. I think since we are developing library module, we should just import from torch.jit, seeing this try except in library code doesn't feel nice